knowledge-base/system-methodaccessexception.md

```diff
@@ -0,0 +1,46 @@
+---
+title: System.MethodAccessException - Attempt by method 'System.Web.Http.HttpConfiguration..ctor(System.Web.Http.HttpRouteCollection)' to access method 'System.Web.Http.HttpConfiguration.DefaultFormatters()' failed
+description: System.MethodAccessException - Attempt by method 'System.Web.Http.HttpConfiguration..ctor(System.Web.Http.HttpRouteCollection)' to access method 'System.Web.Http.HttpConfiguration.DefaultFormatters()' failed.
+type: how-to
+page_title: System.MethodAccessException -  Attempt by method 'System.Web.Http.HttpConfiguration..ctor(System.Web.Http.HttpRouteCollection)' to access method 'System.Web.Http.HttpConfiguration.DefaultFormatters()' failed
+slug: system-methodaccessexception
+res_type: kb
+---
+
+## Description
+
+*System.MethodAccessException: Attempt by method 'System.Web.Http.HttpConfiguration..ctor(System.Web.Http.HttpRouteCollection)' to access method 'System.Web.Http.HttpConfiguration.DefaultFormatters()' failed.*
+
+The above error appears when running a web application with the HTML5 Report Viewer.  
```

Comment:
This is a cause/possible cause and it should go in that section. Add a description about how and when the error appears. Provide sample scenario - e.g. in what projects, which technology, with which viewer, etc.